src/nextjournal/clerk/viewer.cljc

```diff
@@ -629,19 +620,40 @@
     {:nextjournal/render-opts (-> cell (select-keys [:loc]) (assoc :id (processed-block-id (str id "-code"))))}
     (dissoc cell ::doc :result)))
 
+(defn maybe-wrap-var-from-def [val form]
+  (cond->> val
+    (and (var? val) (list? form) (#{'def 'defonce} (first form)))
+    (hash-map :nextjournal.clerk/var-from-def)))
```

Comment:
The analyzer has a slightly different meaning of `deflike?`, can we unify this?

Reply:
yes, I moved `deflike?` to the parser.